.eslintrc.js

```diff
@@ -32,6 +11,14 @@ module.exports = {
       },
     },
   },
+  overrides: [
```

Comment:
@NSeydoux this is what I was thinking for the e2e tests over on #184 — I've not actually done this for the e2e though, but we could.

Reply:
Ah right, it's cleaner than individually muting the offending imports. I'm still a bit puzzled why this is required at all, but it is an improvement.